spec/spellchecker-spec.ts

```diff
@@ -43,19 +43,18 @@ ifdescribe(features.isBuiltinSpellCheckerEnabled())('spellchecker', function ()
   }
 
   // Setup a server to download hunspell dictionary.
-  const server = http.createServer((req, res) => {
+  const server = http.createServer(async (req, res) => {
     // The provided is minimal dict for testing only, full list of words can
     // be found at src/third_party/hunspell_dictionaries/xx_XX.dic.
-    fs.readFile(path.join(__dirname, '/../../third_party/hunspell_dictionaries/xx-XX-3-0.bdic'), function (err, data) {
-      if (err) {
-        console.error('Failed to read dictionary file');
-        res.writeHead(404);
-        res.end(JSON.stringify(err));
-        return;
-      }
+    try {
+      const data = fs.readFile(path.join(__dirname, '/../../third_party/hunspell_dictionaries/xx-XX-3-0.bdic'));
```

Comment:
If we're using `fs.promises` here we need to `await` the call to `fs.readFile`

Reply:
missed that, thanks!